varya/assets/sass/blocks/heading/_style.scss

```diff
@@ -16,35 +16,35 @@ h6, .h6 {
 h1, .h1 {
 	font-size: var(--heading--font-size-h1);
 	letter-spacing: var(--heading--letter-spacing-h1);
-	line-height: var(--wp--typography--line-height, --heading--line-height);
+	line-height: var(--heading--line-height-h1, --global--line-height-body);
```

Comment:
I tested this against GB master thinking it wouldn't work with the new line-height controls. But it does, because they switched to inline style overrides. 

Since the line-height control doesn't rely on `--wp-typography--line-height` variable anymore, what purpose does keeping the fallback (`--global--line-height-body`) serve?

Reply:
Maybe none. 🤔 I'll remove.